civihr_default_theme/scss/base/_custom.scss

```diff
@@ -258,6 +259,10 @@ body {
   width: $width;
 }
 
+.views-exposed-form label {
```

Comment:
shouldn't it be 
```scss
views-exposed-form
  label {
```
isn't there a `.views-exposed-form` already available where you can put the `label` style?

Reply:
Yes right, fixed. But there is no other `.views-exposed-form` available. Also moved the code inside `civihr_employee_portal_theme/civihr_default_theme/scss/drupal/_form.scss`.